CHANGELOG-developer.next.asciidoc

```diff
@@ -199,6 +199,7 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 - Added `ignore_empty_values` flag in `decode_cef` Filebeat processor. {pull}40268[40268]
 - Bump version of elastic/toutoumomoma to remove internal forks of stdlib debug packages. {pull}40325[40325]
 - Refactor x-pack/filebeat/input/websocket for generalisation. {pull}40308[40308]
+- Added support for hyphens in extension keys in `decode_cef` Filebeat processor. {pull}40427[40427]
```

Comment:
This one isn't needed. Can you revert it please. The intent behind this file is to signal changes about the project tooling or important API changes. The addition to CHANGELOG.next.asciidoc is all this change needs.